junit/src/main/java/org/kaazing/k3po/junit/annotation/ScriptProperty.java

```diff
@@ -23,13 +23,14 @@
 /**
  * {@code OverrideProperty} annotation overrides a property in the script.
  * Use format "key \"value\"", so property location "ws://localhost would be
- * @OverrideProperty( "location \"ws://localhost\"" )
+ * \@OverrideProperty( "location \"ws://localhost\"" )
```

Comment:
This should read `ScriptProperty` instead of `OverrideProperty`, right?
Also, if the example being used is a WebSocket URL, then we need a trailing `/` to make it valid.